lib/xandra.ex

```diff
@@ -90,6 +90,20 @@ defmodule Xandra do
   be configured through a subset of the options accepted by
   `start_link/2`. These options are described in the documentation for
   `DBConnection.start_link/2`.
+
+  ## Compression
+
+  Xandra supports compression. To inform the Cassandra server that the
+  connections you start should use compression in the protocol, you can pass the
```

Comment:
It can be confusing with compression of stored data. I want to be clear that this compression is about the transmitted data.